### PR TITLE
Eth rollup develop solo acc20

### PR DESCRIPTION
**Pull request by iStrike7**

### CODE REVIEW

File: rollup/runtime/src/runtime_config.rs [hunk 1]
```diff
@@ -1087,28 +1089,26 @@ pub mod config {
 		impl<T: frame_system::Config> Get<Vec<T::AccountId>> for FoundationAccountsProvider<T> {
 			fn get() -> Vec<T::AccountId> {
 				let accounts: Vec<_> = [
+					// TODO AccountId20
```
Comment (goncer) [inline rollup/runtime/src/runtime_config.rs:1092]: this //TODO still apply?

**Review by goncer — APPROVED** [review]

**Review by mateuszaaa — CHANGES_REQUESTED** [review]
LGTM but let me check how it affects metamask signing

File: rollup/node/src/chain_spec.rs [hunk 2]
```diff
@@ -5,12 +5,20 @@ use rollup_runtime::{
...
 use sc_service::ChainType;
 use sp_consensus_grandpa::AuthorityId as GrandpaId;
-use sp_core::{sr25519, ByteArray, Encode, Pair, Public};
+use sp_core::{ecdsa, ByteArray, Encode, Pair, Public};
 use sp_runtime::{
 	traits::{IdentifyAccount, Verify},
 	BoundedVec,
 };
 
+// ETH Account Info
```
Comment (mateuszaaa) [inline rollup/node/src/chain_spec.rs]: how about setting initial collators & sudo to `Alith` and `Bobth` that are configured by default in polkadot ui??

**Review by mateuszaaa — CHANGES_REQUESTED** [review]
one comment

File: rollup/runtime/src/runtime_config.rs (hunk 1 above)
Comment (iStrike7) [reply rollup/runtime/src/runtime_config.rs:1092]: yes still applies

File: rollup/node/src/chain_spec.rs (hunk 2 above)
Comment (iStrike7) [reply rollup/node/src/chain_spec.rs]: Done

**Pull request merged**